staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -250,14 +250,18 @@ type ObjectMeta struct {
 	// +optional
 	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
 
-	// OwnerClaims is a map of workflow-id to last applied
-	// configuration. A workflow can be the user's name, a
+	// ManagedFields is a map of workflow-id to the set of fields
+	// that are managed by that workflow. This is mostly for internal
+	// housekeeping, and users typically shouldn't need to set or
+	// understand this field. A workflow can be the user's name, a
 	// controller's name, or the name of a specific apply path like
-	// "ci-cd". It keeps track of the ownership of fields. The last
-	// applied configuration is always in the same version as the
-	// parent object. It's used to keep track of the intent of the
-	// workflow-id that submitted that configuration. TODO
-	OwnerClaims map[string]VersionedFieldSet `json:"ownerClaims,omitempty" protobuf:"bytes,17,rep,name=ownerClaims"`
+	// "ci-cd". It keeps track of the ownership of fields. The set
+	// of fields is always in the version the the workflow used when
+	// modifying the parent object. Each workflow-id can only map to a
+	// single version, so if one workflow needs to modify a resource
+	// using multiple versions of the API, it will be assigned multiple
```

Comment:
This last sentence might be too much detail :)

Reply:
I think this was added because of this suggestion https://github.com/kubernetes/kubernetes/pull/69702#discussion_r224617818

@apelisse wdyt?

Reply:
No strong opinion